app/presenters/nsm/check_answers/application_status_card.rb

```diff
@@ -122,12 +122,13 @@ def update_claim
         ].compact
       end
 
-      # TODO: CRM457-1312 - allow below methods are pending adjustment data and comments being available
       def response
-        @response ||= begin
-          response_text = "Fake LAA Response\nSecond line"
-          status.submitted? ? [] : response_text.split("\n").map { sanitize(_1) }
-        end
+        @response ||= if status.granted?
+                        [I18n.t('nsm.steps.view_claim.granted_response')]
+                      else
+                        response_text = claim.assessment_comment
+                        status.submitted? ? [] : response_text.split("\n").map { sanitize(_1) }
```

Comment:
you may be able to use [simple_format](https://crm457-1381-amend-nscc-provider-dev.cloud-platform.service.justice.gov.uk/) here and not decontruct response on calling (at line 32)
```suggestion
                        status.submitted? ? '' : simple_format(response_text)
```

Reply:
using simple_format messes with how the text is combined in the join_strings method so going to leave this as it is for now